browser/src/control/Control.UIManager.js

```diff
@@ -1059,6 +1059,10 @@ L.Control.UIManager = L.Control.extend({
 			this.refreshNotebookbar();
 		else
 			this.refreshMenubar();
+
+		if (typeof window.initializedUI === 'function') {
+			window.initializedUI();
```

Comment:
I see it is repeated in Socket.js in reconnecting. Can we put it in a method here and share (name also will tell others what it does). Like `refreshTheme()` to be unified with other `refresh` name usage .

Reply:
also repeated https://github.com/CollaboraOnline/online/blob/3e8375319361f68d08bd4089536030719183b5fb/browser/src/control/Control.UIManager.js#L645

Reply:
and https://github.com/CollaboraOnline/online/blob/3e8375319361f68d08bd4089536030719183b5fb/browser/src/control/Control.UIManager.js#L414